geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -70,7 +70,7 @@ int calculateSizeOfFieldValuePair(byte[] member, byte[] score) {
 
     while (iterator.hasNext()) {
       byte[] score = iterator.next();
-      validateScoreIsDouble(score);
+      processIncrement(score);
```

Comment:
After this is processed, we need to set score equal to the value that gets returned from `processIncrement` like we do in `zadd`:

```
byte[] score =
          Coder.doubleToBytes(processIncrement(score));
```